SU-SWS/stanford_paragraph_types
# EARTH-89: Added section header to viewfield paragraph type.

# READY FOR REVIEW

# Summary
- What it says on the tin

# Needed By (Date)
- Wednesday EOD

# Urgency
- Moderate

# Steps to Test

1. Check out this branch
2. Revert the feature
3. Edit a view field
4. Add a section header and play with the variants.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
